packages/block-editor/src/components/block-patterns-list/stories/index.story.js

```diff
@@ -0,0 +1,81 @@
+/**
+ * External dependencies
+ */
+import blockLibraryStyles from '!!raw-loader!../../../../../block-library/build-style/style.css';
+
+/**
+ * WordPress dependencies
+ */
+import { useAsyncList } from '@wordpress/compose';
+
+/**
+ * Internal dependencies
+ */
+import BlockPatternsList from '../';
+import { ExperimentalBlockEditorProvider } from '../../provider';
+import patterns from './fixtures';
+
+const blockEditorSettings = {
+	styles: [ { css: blockLibraryStyles } ],
+};
+
+const meta = {
+	component: BlockPatternsList,
+	title: 'BlockEditor/BlockPatternsList',
+	argTypes: {
+		blockPatterns: { description: 'The patterns to render.' },
+		shownPatterns: {
+			description:
+				'Usually this component is used with `useAsyncList` for performance reasons and you should provide the returned list from that hook. Alternatively it should have the same value with `blockPatterns`.',
+		},
+		showTitle: {
+			description: 'Whether to render the title of each pattern.',
+			table: {
+				defaultValue: { summary: true },
+				type: { summary: 'boolean' },
+			},
+		},
+		onClickPattern: { type: 'function' },
+		onHover: { type: 'function' },
+		showTitlesAsTooltip: {
+			description:
+				'Whether to render the title of each pattern as a tooltip. If enabled, it takes precedence over `showTitle` prop.',
+		},
+		orientation: {
+			description:
+				'Orientation for arrow key navigation. Internally Composite component is used and it"s default value.',
```

Comment:
Not sure about this specific component, but usually we prefer not exposing how a component internally works, so that in the future we have more freedom on changing that aspect. Maybe you could change it to something more generic, like

```suggestion
				'Orientation for the underlying composite widget.',
```